cmd/plume/specs.go

```diff
@@ -207,6 +207,56 @@ var (
 				},
 			},
 		},
+		"developer": channelSpec{
+			BaseURL: "gs://builds.developer.core-os.net/boards",
+			Boards:  []string{"amd64-usr", "arm64-usr"},
+			Destinations: []storageSpec{storageSpec{
+				BaseURL:       "gs://builds.developer.core-os.net/releases",
+				NamedPath:     "current",
+				VersionPath:   true,
+				DirectoryHTML: true,
+				IndexHTML:     true,
+			}},
+			GCE: gceSpec{
+				Project:     "coreos-gce-testing",
+				Family:      "coreos-dev",
+				Description: "CoreOS Container Linux development image",
+				Image:       "coreos_production_gce.tar.gz",
+				Publish:     "coreos_production_gce.txt",
+				Limit:       2,
+			},
+			AWS: awsSpec{
+				BaseName:        "ContainerLinuxDev",
+				BaseDescription: "CoreOS Container Linux development image",
+				Prefix:          "coreos_production_ami_",
+				Image:           "coreos_production_ami_vmdk_image.vmdk.bz2",
+				Partitions: []awsPartitionSpec{
```

Comment:
This complexity (two partitions, three regions) exists to test `plume pre-release`.  For nightly Jenkins builds I think we should use `us-west-2` only.  The `LaunchPermissions` in the second `awsPartitionSpec` refers to `coreosdev` and should probably stay.